res/database/Default/user/kieferer.xml

```diff
@@ -31,14 +37,31 @@ ist wie gedacht. Bald kämpfen sie um Leben und Tod.</de>
 
 		<scripttemplate product="1" licence_type="1" guid="scripttemplate-random-kieferer-mummyattack">
 			<title>
-				<de>Mumien greifen an</de>
-				<en>Mummy Attack</en>
+				<de>%TITLE%</de>
+				<en>%TITLE%</en>
 			</title>
 			<description>
 				<de>Ein ganz normaler Teenagerausflug endet in einem Besäufnis und einem Wald voller Untoter.</de>
 				<en>A normal teenage trip ends in drunkenness and a forest full of undead.</en>
 			</description>
-
+			<variables>
+				<title>
+					<de>%WHO% greifen an|Angriff der %WHO%|Rache der %WHO%|Umzingelt von %WHOD%|Wald der %WHO%</de>
+					<en>%WHO% Attack|Assault of the %WHO%|Revenge of the %WHO%|Surrounded by %WHOD%|Forest of the %WHO%</en>
+				</title>
+				<whoi>
+					<de>Mumien|Zombies</de>
+					<en>Mummies|Zombies</en>
+				</whoi>
+				<who>
+					<de>%WHOI%|Vampiere|Geister|Monster|Werwölfe</de>
```

Comment:
Vampire - auch in deutsch mit "kurzem i"

Reply:
wie peinlich